msteams-platform/resources/team-chat-member-api-changes.md

```diff
@@ -8,6 +8,9 @@ ms.author: ojchoudh
 ---
 # Changes to Teams Bot APIs for Fetching Team/Chat Members
 
+>[!NOTE]
+> We've started with deprecation process for `TeamsInfo.getMembers` and `TeamsInfo.GetMembersAsync`. Initially, they will be throttled to 5 requests per minute and return a maximum of 10K members per team. This can result in the full roster not being returned as the team size [increases](https://blogs.microsoft.com/2020/09/22/whats-new-in-microsoft-teams-microsoft-ignite-2020/). Please refer to the [steps below](https://docs.microsoft.com/microsoftteams/platform/resources/team-chat-member-api-changes#api-changes) and update to version 4.8 (or above) of the Bot Framework SDK and switch to the paginated API endpoints or the `TeamsInfo.GetMemberAsync` single user API.
```

Comment:
Suggest we keep the first two sentences of this note same as that of the previous note.
```suggestion
>[!NOTE]
> We've started with deprecation process for `TeamsInfo.getMembers` and `TeamsInfo.GetMembersAsync`. Initially, they will be throttled to 5 requests per minute and return a maximum of 10K members per team. This can result in the full roster not being returned as the team size [increases](https://blogs.microsoft.com/2020/09/22/whats-new-in-microsoft-teams-microsoft-ignite-2020/). To view the list of upcoming changes, update to version 4.8 or higher, and to switch to the paginated API endpoints or the `TeamsInfo.GetMemberAsync` single user API, see [API changes](https://docs.microsoft.com/microsoftteams/platform/resources/team-chat-member-api-changes#api-changes). 
```

Reply:
I'm okay with this framing if it's all the same to you. We need to explicit in this page.